index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title></title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/6776bca25f.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="first_section_div">
+      <header>
+        <nav class="nav_bar">
+          <div class="name_logo" id="name_tag">
+            <a href="index.html">RAPHEAL OKONTA</a>
+          </div>
+          <i class="fa-solid fa-bars navitems fa-2xl"></i>
+        </nav>
+      </header>
+      <div class id="main_section">
+        <div id="sub_section1">
+          <div id="intro_tag">
+            <h1 id="intro_name">I’m Rapheal, <br />Glad to see <br />you!</h1>
+            <br />
+            <h2 class="intro_text">I am a Software developer. <br /></h2>
+          </div>
+          <div id="intro_page">
+            <p id="introduction">
+              I can help you build a product , feature or website.<br />
+              Look through some of my work and experience!<br />
+              If you like what you see and have a project you need coded,
+              <br />don’t hestiate to contact me.
+            </p>
+          </div>
+        </div>
+        <div>
+          <img class="side_image" src="images/portfolio_image.png.png" />
+        </div>
+        <section id="sub_section2">
+          <p class="social_media">LET'S CONNECT</p>
+          <ul id="social_media_tag">
+            <li class="social_media_link">
+              <a href="https://www.linkedin.com/"
+                ><i class="colored fa-brands fa-linkedin-in fa-xl"></i
+              ></a>
+            </li>
+            <li class="social_media_link">
+              <a href="https://twitter.com/home"
+                ><i class="colored fa-brands fa-twitter fa-xl"></i
+              ></a>
+            </li>
+            <li class="social_media_link">
+              <a href="https://medium.com/"
+                ><i class="colored fa-solid fa-m fa-xl"></i
+              ></a>
+            </li>
+            <li class="social_media_link">
+              <a href="https://github.com/"
+                ><i class="colored fa-brands fa-github fa-xl"></i
+              ></a>
+            </li>
+            <li class="social_media_link">
+              <a href="https://www.angellist.com/"
+                ><i class="colored fa-solid fa-hand-peace fa-xl"></i
+              ></a>
+            </li>
+          </ul>
+        </section>
+      </div>
+    </div>
```

Comment:
- [ ] Kindly wrap this headline container into a `section` tag. Generally, the `section` represents a generic standalone section of the document, so kindly use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

Reply:
@levy002,
Thanks for your review. I will effect the changes asap. But I need clarity, when you say headline container, are you referring to lines 25-40?